homeassistant/components/wiser/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+  "domain": "wiser",
+  "name": "Wiser Heating Component for Home Assistant",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/wiser",
+  "dependencies": [],
+  "codeowners": [
+    "@asantaga",
+    "@msp1974"
+  ],
+  "requirements": [
+    "wiser-heating-api==1.0.8"
+  ],
+  "zeroconf": [
+    "_http._tcp.local."
```

Comment:
We should not use this type of general discovery. The project is looking for methods to provide more active filtering on this, however, that is not currently the case.

Reply:
removed and removed from config_flow